packages/config/config/manufacturers.json

```diff
@@ -402,7 +402,7 @@
 	"0x0198": "Venstar Inc.",
 	"0x0199": "Wireless Maingate AB",
 	"0x019a": "Sensative AB",
-	"0x019b": "ThermoFloor",
+	"0x019b": "Heatit Controls AB",
```

Comment:
The most recent specification package lists "ThermoFloor" (formerly known as Heatit) for this ID.

Reply:
We should probably keep this as is.

Reply:
Yes, this is confirmed a with them about a week ago.

Thermofloor owns Heatit so that´s probally why it´s listed as thermofloor in the offical docs.
And they started with their devices as thermofloor.